akka-docs/src/main/paradox/stream/stream-dynamic.md

```diff
@@ -206,7 +206,7 @@ i.e. `int` greater than or equal to 0 and less than number of consumers.
 
 The resulting `Source` can be materialized any number of times, each materialization effectively attaching
 a new consumer. If there are no consumers attached to this hub then it will not drop any elements but instead
-backpressure the upstream producer until consumers arrive. This behavior can be tweaked by using the combinators
+backpressure the upstream producer until consumers arrive. This behavior can be tweaked by using the operators
```

Comment:
The plural here is fishy, actually the whole sentence feels a bit strange.

What about: "This behavior can be tweaked by using an operator, for example `.buffer` with a drop strategy,"